tests/attr/test_common.py

```diff
@@ -37,43 +31,3 @@ def test_validate_nt_type(self) -> None:
         _validate_noise_tunnel_type("smoothgrad", SUPPORTED_NOISE_TUNNEL_TYPES)
         _validate_noise_tunnel_type("smoothgrad_sq", SUPPORTED_NOISE_TUNNEL_TYPES)
         _validate_noise_tunnel_type("vargrad", SUPPORTED_NOISE_TUNNEL_TYPES)
-
-    def test_select_target_2d(self) -> None:
```

Comment:
It looks like these tests are being deleted, could we possibly move them to a test class corresponding to the generic utils?

Reply:
That one I moved to `test/utils/test_common.py`. It looks like I forgot to push it into PR